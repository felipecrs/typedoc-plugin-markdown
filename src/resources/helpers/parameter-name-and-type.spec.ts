import * as Handlebars from 'handlebars';

import { setup } from '../../../test/utils';

describe(`[helpers] parameterNameAndType`, () => {
  let project: any;

  beforeAll(() => {
    ({ project } = setup());
  });

  test(`sould compile`, () => {
    const data = project.findReflectionByName('objectLiteral');
    const result = Handlebars.helpers.parameterNameAndType.call(data);
    expect(result).toMatchSnapshot();
  });
});
