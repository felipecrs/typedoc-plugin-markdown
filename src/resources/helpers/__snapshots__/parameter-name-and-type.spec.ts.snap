// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[helpers] parameterNameAndType sould compile 1`] = `"â–ª\`Const\`  **objectLiteral**: *object*"`;
